linter/.golangci-maprange.yml

```diff
@@ -0,0 +1,25 @@
+run:
+  timeout: 5m
+
+linters:
+  disable-all: true
+  # Enable specific linter
+  # https://golangci-lint.run/usage/linters/#enabled-by-default-linters
+  enable:
+    - maprange
+
+linters-settings:
+  custom:
+    maprange:
+      path: bin/maprange.so
+      description: "Detects range loops over maps"
+
+issues:      
+  max-issues-per-linter: 10000
+  max-same-issues: 10000
+
+  exclude-files:
+    - ".*\\.pb\\.go"
+    - ".*\\.pb\\.gw\\.go"
+    - ".*\\.pulsar\\.go"
+    - ".*_mocks\\.go"
```

Comment:
should you exclude the linter dir itself so you can keep your test?

Reply:
It is not reporting anything on my tests, which is strange. Investigating why, golangci automatically excludes these dirs:
vendor$, third_party$, testdata$, examples$, Godeps$, builtin$